backend/api/groups.py

```diff
@@ -0,0 +1,29 @@
+# backend/api/groups.py
+from fastapi import APIRouter, HTTPException
+from starlette import status
+from bson import ObjectId
+
+from backend.api.api_request_classes import CreateGroupRequest
```

Comment:
You should import the whole module, not just the required class. E.g., `from backend.api import api_request_classes as api_req`